vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/BigDecimalField.java

```diff
@@ -385,9 +392,20 @@ public BigDecimal getValue() {
      */
     @Override
     protected void validate() {
-        var requiredValidation = ValidationUtil.checkRequired(required,
```

Comment:
The `required` field is now unused (never read) and can be removed:
https://github.com/vaadin/flow-components/blob/563c1b8be95948afda96bdd7356fff4e7cd65482/vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/BigDecimalField.java#L78

Reply:
Done.